spec/models/active_job_log/job_spec.rb

```diff
@@ -5,10 +5,6 @@
     expect(create(:active_job_log_job)).to be_valid
   end
 
-  describe "Validations" do
-    it { is_expected.to enumerize(:status).in(:queued, :pending, :finished, :failed) }
-  end
```

Comment:
por qué se sacó esto?

Reply:
Buena pregunta. Al actualizar la versión de enumerize dejó de incluirse el matcher de RSpec. No me preocupé porque la verdad medio inútil el test pero bueno, viendo que te llamó la atención aproveché para investigar y resulta que se puede agregar a mano y vuelve a funcionar https://github.com/platanus/active_job_log/pull/72/files#diff-053aa3ea46cb7c6a649e0d0fc592ce729b66e1d44be475516fd58857ba695862R28